### ISSUE TITLE
Not returning unless I open app on phone

### ISSUE THREAD

**Issue by CAT1210** [bdr99/pymazda#23]
Recently, I have started having an issue with this. This Python script will not return anything and will just run and run without returning any results. The only way to get it to work again is to open the Mazda app on my phone, sign in, then it will work well again for a little while. Any ideas?

**Comment by bdr99** [comment]
Hi @CAT1210, sorry I didn't get a chance to reply to this issue. I haven't seen this problem before. I'm curious if you were you able to resolve it?

**Comment by CAT1210** [comment]
I think my application is different from most. I developed an application that takes it's information from results of PyMazda every few minutes. So, my PyMazda Python script would run every few minutes to get the latest data from my vehicle. I think what was happening is the PyMazda script was not getting data at times and returning anything (probably something on the Mazda cloud side), the PyMazda script didn't know this and would keep trying to run again, even though the it was still choking from not getting data. I think it was causing a loop that the Mazda servers didn't like and the way that it could be broken was logging into the MyMazda app on my phone (another device). So, all I did, was change the "await asyncio.wait(client.get_vehicles)" part of the PyMazda script to "await ayncio.wait_for(client.get_vehicles(), timeout=20)" so the call would fail if nothing is returned within 20 seconds. I can look at results and see that this happens sometimes, no results are return, the script times out, but the next call works just fine.
